src/DataTable/TableRow.js

```diff
@@ -70,6 +70,16 @@ class TableRow extends PureComponent {
     expanded: false,
   }
 
+  componentDidMount() {
+    const { row } = this.props;
+    const { defaultExpandedField } = this.context;
+    if (defaultExpandedField) {
```

Comment:
super minor nitpick ;) put a line a break between if statements and variables declarations

Reply:
Done